README.md

```diff
@@ -212,13 +203,12 @@ We welcome community contributions to nGraph. If you have an idea for how to
 improve it:
 
 * Share your proposal via [GitHub issues].
-* Make sure your patch is in line with Google style by setting up your git `pre-commit` hooks.  First, ensure `clang-format` is in your path, then:
-
-   ```
-   pip install pre-commit autopep8 pylint
-   pre-commit install
-   ```
-
+* Make sure your patch is in line with Google style by setting up your git 
```

Comment:
This is not OSX specific. Either we need to move this to common section or just remove it.

Reply:
This heading is not under OS X, it is a higher level heading, should be applicable to all options.  I understand what you're saying, though.  Let me see what else makes sense.